app/frontend/wallet/shelley/build-transaction.ts

```diff
@@ -42,7 +49,7 @@ export function computeTxPlan(
   inputs: Array<Input>,
   outputs: Array<Output>,
   possibleChange?: Output,
-  cert?: any
+  cert?: any // TODO: cert
```

Comment:
can't we use "Cert" type here as well? It seems as if we were just passing that parameter to the TxPlan return value anyway